checkpoint-multi-nic/azuredeploy.json

```diff
@@ -0,0 +1,449 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "outputs": {
+        "PublicIPAddress": {
+            "type": "string",
+            "value": "[reference(variables('publicIPAddressID')).IpAddress]"
+        }
+    },
+    "parameters": {
+        "adminPassword": {
+            "metadata": {
+                "description": "Administrator password"
+            },
+            "type": "securestring"
+        },
+        "authenticationType": {
+            "allowedValues": [
+                "password",
+                "sshPublicKey"
+            ],
+            "defaultValue": "password",
+            "metadata": {
+                "description": "Authentication type"
+            },
+            "type": "string"
+        },
+        "newStorageAccountName": {
+            "metadata": {
+                "description": "Name of storage account"
+            },
+            "type": "string"
+        },
+        "offer": {
+            "allowedValues": [
+                "R7710BYOL",
+                "R7710PAYG"
+            ],
+            "defaultValue": "R7710BYOL",
+            "metadata": {
+                "description": "Offer version and type"
+            },
+            "type": "string"
+        },
+        "sshPublicKey": {
+            "defaultValue": "",
+            "metadata": {
+                "description": "Administrator SSH public key"
+            },
+            "type": "string"
+        },
+        "vmName": {
+            "metadata": {
+                "description": "Name of the Security Gateway"
+            },
+            "type": "string"
+        },
+        "vmSize": {
+            "defaultValue": "Standard_D3",
+            "metadata": {
+                "description": "Size of the VM"
+            },
+            "type": "string"
+        }
+    },
+    "resources": [
+        {
+            "apiVersion": "2015-05-01-preview",
```

Comment:
I suggest making api version a variable, it'll be easier to move to newer versions for resources. Also suggest using the latest API version 2016-06-15